.github/workflows/stale.yml

```diff
@@ -20,11 +20,11 @@ jobs:
           exempt-all-assignees: false
           exempt-draft-pr: true
           stale-issue-label: "Pending Closure"
-          stale-pr-label: '0 - Waiting on User'
-          only-pr-labels: '0 - Waiting on User'
+          stale-pr-label: 'Pending Closure'
+          only-labels: '0 - Waiting on User'
           close-issue-label: "No Response / Stale"
           close-pr-label: "No Response / Stale"
-          exempt-issue-labels: 'Security / CVE,0 - Backlog,1 - Ready for work,2 - Working,3 - Review,4 - Done,5 - Push required'

```

Comment:
Do we want to remove these labels as exempting labels?

There could be cases where we need more information from the user, but we know it will need to be looked at. If we have such a case, we can use these labels to prevent the issue from being closed even though it is waiting for the user.

Reply:
I feel that if we have added the 'waiting for user' label, we cannot proceed with the issue until they come back. If we want to keep the issue open regardless of whether the user comes back or not, we shouldn't add that label at all.

That is something we could add as a process.

Reply:
Fair enough.